### ISSUE TITLE
Add rendering for leisure=horse_riding

### ISSUE THREAD

**Issue by gscscnd** [gravitystorm/openstreetmap-carto#2344]
Could you add rendering for the [leisure=horse_riding](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dhorse_riding) tag?

Example: [323603669](https://www.openstreetmap.org/way/323603669).

**Comment by kocio-pl** [comment]
I guess this could be rendered similar to dog park with horseshoes instead of paws or with a solid background and a horseshoe icon (like in playground).

**Comment by HolgerJeromin** [comment]
Why do riders need another tag (used 3000 times) ? 
The wiki page does not explain why sport=equestrian with sports centre (used 2000 times)  is not enough. 

With this problem I am not sure if we should render it special.

**Comment by kocio-pl** [comment]
Maybe the difference is between sport and leisure? This is not intended for horse racing, for example.

**Comment by gscscnd** [comment]
@kocio-pl, that’s what I think (but please note that there’s separate sport=horse_racing tag).

I mean, we can go to leisure=horse_riding to learn riding basics and practise (spare time activity). And in sports centre we can take part in riding competitions (professional activity). While it may be the same for a small riding stable, I think there can be a clear separation for bigger ones:

* [431355912 — there you learn and take care of horses](https://www.openstreetmap.org/way/431355912)
* [343063112 — and that’s nearby sports centre](https://www.openstreetmap.org/way/343063112)

Wiki [(Riding)](https://wiki.openstreetmap.org/wiki/Riding) doesn’t tell what is the exact difference between sport=equestrian (+ some sports centre tag) and leisure=horse_riding (just ‘exercise’ versus ‘practise-in-spare-time’), but here I’ve written what I think about it.

**Comment by HolgerJeromin** [comment]
We don't have such a difference for any other sport, do we?

**Comment by kocio-pl** [comment]
I think the wiki descriptions for both tags are just too short to know. 

There's some clue on a [Riding page](http://wiki.openstreetmap.org/wiki/Riding): 
- leisure=pitch + sport=equestrian is for "riding ground" (A place to exercise horseback riding.) 
- leisure=horse_riding is for "riding stable" (A place where people go in their spare time and practice horseback riding.)

but my English is not good enough to know what it really means.

**Comment by gscscnd** [comment]
One more thing I’ve just read on [German wiki](https://wiki.openstreetmap.org/wiki/DE:Tag:leisure%3Dhorse_riding): leisure=pitch with sport=equestrian is considered to be a fragment/subarea of leisure=horse_riding. In other words, leisure=horse_riding includes all facilities like stables (buildings), buildings where you practise riding (‘Riding hall’) _and_ leisure=pitch if such exists.

**Comment by kocio-pl** [comment]
BTW: the mentioned combination leisure=sports_centre + sport=equestrian (2k of uses) does not have its own definition (at least I didn't find it) and is less frequent than leisure=pitch + sport=equestrian (14k of uses).

I guess equestrian tags need some work before we can render them. Please use Tagging list probably to discuss it and make wiki definitions more clear.

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 14 set 2016, alle ore 11:40, gscscnd <notifications@github.com> ha scritto:
> 
> Wiki (Riding) doesn’t tell what is the exact difference between sport=equestrian (+ some sports centre tag) and leisure=horse_riding (just ‘exercise’ versus ‘practise-in-spare-time’), but here I’ve written what I think about it.


clearly there are several quite different leisure things and sports you can do with horses, like horse racing, rent horses for people "going for a walk", halls for kids to go riding, people jumping with horses, etc. and rendering them differently can be justified.

**Comment by HolgerJeromin** [comment]
After my research the leisure=horse_riding should be the complete outline containing multiple buildings, pitches and so on. So it is equivalent to a leisure=sport_centre which contains buildings and pitches, too.

**Comment by Hufkratzer** [comment]
... and equivalent to a leisure=golf_course. Why render leisure=golf_course and leisure=horse_riding not?

(Here is a good photo of auch a leisure=horse_riding area:
https://commons.wikimedia.org/wiki/File:Warendorf,_Reitanlage_Josephshof_--_2014_--_8591.jpg?uselang=de )

**Comment by HolgerJeromin** [comment]
Golf is an interesting example.
Leisure = golf course is probably a pitch and no sports centre.

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 15 set 2016, alle ore 23:00, Holger Jeromin <notifications@github.com> ha scritto:
> 
> Golf is an interesting example.
> Leisure = golf course is probably a pitch and no sports centre.


-1, we are looking here how people do tag, not discussing how they should. A leisure=golf_course comprises more than just the 9/18 holes course, typically people use it for the whole area, possibly comprising driving range, putting area, parking, club house etc.

**Comment by jojo4u** [comment]
sport_centre is for venues with a **range** of sports (exception: natatorium). For venues with only one sport an own tag should be used when possible.

**Comment by HolgerJeromin** [comment]
I often surround many tennis courts with a sports centre which gets the name and address.

**Comment by Hufkratzer** [comment]
This is probably just because the wiki **strongly discourages** to use **leisure=tennis**, see
http://wiki.openstreetmap.org/wiki/Tag:sport=tennis (scroll down)

For horse_riding it is the opposite: The wiki clearly **recommends** to use **leisure=horse_riding**, see
http://wiki.openstreetmap.org/wiki/Riding ("Riding stable")
This is probably because not everyone sees / practices horse riding as a sport, see
http://en.wikipedia.org/wiki/Pleasure_riding

So in the wiki horse_riding is handled more like golf and not like tennis, compare
http://wiki.openstreetmap.org/wiki/Tag:sport=golf

Maybe you can render leisure=sports_centre + sport=* too if it is used for just one kind of sport?

**Comment by mboeringa** [comment]
> sport_centre is for venues with a range of sports (exception: natatorium). For venues with only one sport an own tag should be used when possible.

I don't agree. The distinction of what is a single sport or **leisure=sports_centre** is often very blurry. Even for sports_centres that may apparently be easy to distinguish as belonging to a single "sport", that sport may have varieties being regularly played on the same pitches, or on another type or part of the pitch or another facility within the perimeters. E.g. soccer or field_hockey may switch to indoor varieties in winter months.

**Comment by kocio-pl** [comment]
Please check if this discussion doesn't belong to Tagging list, since we have too many questions regarding tagging before we can even start talking about rendering. There's already a thread started by @gscscnd:

https://lists.openstreetmap.org/pipermail/tagging/2016-September/030134.html

**Comment by Tomasz-W** [comment]
What about this issue future? Is an icon project needed here?

**Comment by kocio-pl** [comment]
I don't remember the output of the Tagging discussion, you should probably check it first.

**Comment by Hufkratzer** [comment]
If you need an icon perhaps you can take one of the two possibilties from https://github.com/gravitystorm/openstreetmap-carto/issues/844: ![hhead14-1](https://cloud.githubusercontent.com/assets/5958184/22512328/eed19da6-e898-11e6-80b7-78c50006b500.png) or ![hshoe-14](https://cloud.githubusercontent.com/assets/5958184/22522126/c872539a-e8ba-11e6-912d-86bd2076643b.png)

**Comment by Tomasz-W** [comment]
> I don't remember the output of the Tagging discussion, you should probably check it first.

The output was to stay with this tag, and use `sport=equestrian` only in combination with `leisure=pitch` to map certain riding areas.

I also thought about an icon with horse jumping above obstacle. What do you think?

**Comment by Hufkratzer** [comment]
JOSM uses these icons:
![equestr15](https://user-images.githubusercontent.com/5958184/34885860-8ebcf3fa-f7c1-11e7-959d-6c80a2bfa6f5.png) for sport=equestrian (+pitch) [(equestrian.svg)](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/sport/equestrian.svg)
![horse_riding](https://user-images.githubusercontent.com/5958184/34885824-74caca4e-f7c1-11e7-9f6f-300d4c3defe1.png) for leisure=horse_riding [(horse_riding.svg)](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/leisure/horse_riding.svg)  

I think this makes sense,
![equestr15](https://user-images.githubusercontent.com/5958184/34885860-8ebcf3fa-f7c1-11e7-959d-6c80a2bfa6f5.png) looks more sporty, it may even try to symbolize a jumper (compare photo in [wiki](https://wiki.openstreetmap.org/wiki/Tag:sport=equestrian))
![horse_riding](https://user-images.githubusercontent.com/5958184/34885824-74caca4e-f7c1-11e7-9f6f-300d4c3defe1.png) looks more relaxed, symbolizing a [(p)leasure rider](http://en.wikipedia.org/wiki/Pleasure_riding)
So I think the horse jumping above obstacle could better be used for sport=equestrian  (https://github.com/gravitystorm/openstreetmap-carto/issues/844)

**Comment by gscscnd** [comment]
I prefer JOSM’s icons, they’re very clear, at least for me. That Mapbox’s looks like ‘i’ on a horse — it’s caused by those white jodhpurs which additionally make the icon look two­‑colored instead of single­‑colored.

**Comment by Hufkratzer** [comment]
I think the JOSM icons would need refinement before they can be used on the standard map.

Perhaps these similar looking alternatives can be used:
![horsewoman-mit-backgrrgood2](https://user-images.githubusercontent.com/5958184/42904520-9e32c70e-8ad5-11e8-84d7-898f5c2b1dc8.png) for leisure=horse_riding (from [here](https://publicdomainvectors.org/en/free-clipart/Woman-riding-horse-silhouette/68010.html))
![dressur-mit-background7good](https://user-images.githubusercontent.com/5958184/42904177-76bb64d4-8ad4-11e8-80d9-6e40a3fccdad.png) for sport=equestrian (from [here](https://commons.wikimedia.org/wiki/File:Bundesgestüt_Piber_logo.svg), this is dressage and not jumping, but in 14px it looks quite similar; and dressage is sport too)

Compare current JOSM icons:
![horse_riding](https://user-images.githubusercontent.com/5958184/34885824-74caca4e-f7c1-11e7-9f6f-300d4c3defe1.png) [horse_riding.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/leisure/horse_riding.svg)  
![equestr15](https://user-images.githubusercontent.com/5958184/34885860-8ebcf3fa-f7c1-11e7-959d-6c80a2bfa6f5.png) [equestrian.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/sport/equestrian.svg)

**Comment by Tomasz-W** [comment]
@Hufkratzer We can't use this icons due to copyrights. There are also some technical requirements: https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#map-icon-guidelines

**Comment by Hufkratzer** [comment]
AFAICT the icons are in the public domain? What is wrong wirh that?
They are a bit complex, but perhaps it's easier to remove some complexity than to draw new icons..

**Comment by Tomasz-W** [comment]
@Hufkratzer Can you refine JOSM icons due to your remarks?

**Comment by Hufkratzer** [comment]
Perhaps, I don't know. I am not experienced with icons. It will take time, I assume it is not urgent.

**Comment by Tomasz-W** [comment]
I can try to edit and make those JOSM icons better, but I have to know what is wrong with them in your opinion, beceause for me they look ok.

**Comment by Adamant36** [comment]
@Tomasz-W Any idea where this one is at and what icon we are going with?

**Comment by Hufkratzer** [comment]
I have some drafts meanwhile, but they are not finished yet. (It's not so easy to draw hores and riders correctly.) How urgent is it?

**Comment by Adamant36** [comment]
@Hifkratzer Not urgent at all. Take all the time you want. I just got the impression there was a usable icon up there somewhere in all of them, but I wasnt sure what one. I'm glad someone is working on a different though. Since none of them seemed to really work for me.

**Comment by Hufkratzer** [comment]
Here are **drafts** of how I would draw the horse icons:

![14equestrian](https://user-images.githubusercontent.com/5958184/44619103-9642ae00-a881-11e8-8028-78486643732a.png) [equestrian.svg](https://gist.github.com/Hufkratzer/24b85cd8f5402ee30a23de5c64215d10#file-equestrian-svg)
![14horse_riding](https://user-images.githubusercontent.com/5958184/44619137-fb969f00-a881-11e8-822c-7d22488b4183.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/24b85cd8f5402ee30a23de5c64215d10#file-horse_riding-svg)
Some other icons for comparison
![14equestrianjosm14](https://user-images.githubusercontent.com/5958184/44620070-cc3a5f00-a88e-11e8-9356-feb5c5ab9013.png) JOSM [equestrian.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/sport/equestrian.svg)
![14horse_ridingjosm14](https://user-images.githubusercontent.com/5958184/44620089-19b6cc00-a88f-11e8-9c7f-6bf74bfda01b.png) JOSM [horse_riding.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/leisure/horse_riding.svg)
![14golf](https://user-images.githubusercontent.com/5958184/44619174-7bbd0480-a882-11e8-8e14-8c6e6b05416e.png) [golf.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/golf.svg)
![14handball](https://user-images.githubusercontent.com/5958184/44619175-87a8c680-a882-11e8-8567-5a297c1efdf6.png) [handball.svg](https://gist.github.com/Tomasz-W/a56711ac75baf0bddf8655a04bdf8924#file-sport-handball-svg)
![14picnic](https://user-images.githubusercontent.com/5958184/44619238-81671a00-a883-11e8-922d-2dab9245d39b.png) [picnic.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/picnic.svg)
![14playground](https://user-images.githubusercontent.com/5958184/44619244-9d6abb80-a883-11e8-8adb-3f2247be05dc.png) [playground.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/playground.svg)
![14running](https://user-images.githubusercontent.com/5958184/44619252-c0956b00-a883-11e8-8a52-b5d35501ed27.png) [running.svg](https://gist.github.com/Tomasz-W/a56711ac75baf0bddf8655a04bdf8924#file-sport-running-svg)
![14sauna](https://user-images.githubusercontent.com/5958184/44619271-ec185580-a883-11e8-9e70-8f601754e49e.png) [sauna.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/sauna.svg)
![14sportshop](https://user-images.githubusercontent.com/5958184/44619277-0a7e5100-a884-11e8-834a-ffd7841aeec6.png) [sports.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/shop/sports.svg) (shop)
![14toys](https://user-images.githubusercontent.com/5958184/44619293-4e715600-a884-11e8-8946-1e5eca932baa.png) [toys.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/shop/toys.svg) (shop)
![14veterinary](https://user-images.githubusercontent.com/5958184/44619304-72349c00-a884-11e8-816c-c078b8398975.png) [veterinary.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/veterinary.svg)

Some of them a bit larger (70px):
![70equestrian](https://user-images.githubusercontent.com/5958184/44619323-d9525080-a884-11e8-814a-59858dd41e24.png) 
![70horse_riding](https://user-images.githubusercontent.com/5958184/44619330-ef601100-a884-11e8-9ff2-0fce7acc7245.png)
![70toys](https://user-images.githubusercontent.com/5958184/44619999-e32c8180-a88d-11e8-9765-6f92c2589412.png)
![70running](https://user-images.githubusercontent.com/5958184/44619344-2df5cb80-a885-11e8-8429-412d2a2a4780.png)
![70sportshop](https://user-images.githubusercontent.com/5958184/44619347-36e69d00-a885-11e8-857a-0b63337d6412.png)

### JOSM icons - what was improved? ###
![70equestrianjosm14](https://user-images.githubusercontent.com/5958184/44619389-defc6600-a885-11e8-8539-9a802a5b12f5.png) [equestrian.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/sport/equestrian.svg)
- unclear what they are doing (I assumed they try to jump)
- rider sits too far back on the horse
- legs of horse not correct

Compare other icons:
- https://www.shutterstock.com/es/image-vector/riding-horse-silhouette-equestrian-sport-equipment-482507050
- https://www.shutterstock.com/es/image-vector/overcoming-obstacles-horse-symbol-vector-102902240
- https://www.shutterstock.com/es/image-vector/rider-silhouette-vector-46920352

Compare photos:
- https://commons.wikimedia.org/wiki/File:ILHS_2016_-_20160909_-_Donovan_Ber_et_Virtuose_du_Faubourg_2.jpg
- https://www.dreamstime.com/royalty-free-stock-image-horse-show-jumping-image27446276
- https://static1.squarespace.com/static/52092d15e4b0f89d3271e4d3/t/595f6c4c6b8f5b29ad818b2f/1499425872877/?format=500w

![70horse_ridingjosm14](https://user-images.githubusercontent.com/5958184/44619544-93978700-a888-11e8-8608-0405e30a7326.png) [horse_riding.svg](https://josm.openstreetmap.de/browser/josm/trunk/images/presets/leisure/horse_riding.svg)
- rider sits too far back on the horse
- legs of horse a bit strange
- head of horse too high for pleasure riding

Compare other icons:
- https://publicdomainvectors.org/en/free-clipart/Woman-riding-horse-silhouette/68010.html
- https://www.shutterstock.com/es/image-vector/silhouette-rider-trotting-on-horse-779951746

Compare photos:
- https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJxRLBleliGSOh9wT68boYWxz0IidbUGbwn2UF3sgEwI-PYrrY
- https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTMCnWp9NsJk-zXXLZzw7A8oe6EzssIELqO1y5NLBy8yAeVxPVOUg
- https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS9jq5zU-eTQsGbWbMqj62A-6p4obA4O39tlQdZZ0bj5FOBMTbvdQ

### @Tomasz-W : ###
My [horse icon drafts](https://gist.github.com/Hufkratzer/24b85cd8f5402ee30a23de5c64215d10) were drawn by hand (on paper), scanned and svged automatically in Inkscape, so they are not exact yet, just drafts! I have tried to place at least the hooves and noses of the horses in the center of a pixel. Is this helpful for you to improve the JOSM icons or to create new ones that meet all requirements? Do you need anything else or what do you propose?

**Comment by Tomasz-W** [comment]
@Hufkratzer I made icon designs based on your drafts (they were not useful for OSM at all, we need to have a complete project made in Inkscape from the beginning):
* equestrian
![sport equestrian](https://user-images.githubusercontent.com/25656654/44622047-54c9f700-a8b1-11e8-9a5b-d5ac2f887396.png)
* horse_riding
![leisure horse_riding](https://user-images.githubusercontent.com/25656654/44622049-5b586e80-a8b1-11e8-8502-2d5e93a6e50b.png)

If you want to make this shapes better, I'm putting a raw Inkscape files which you can edit etc.: https://gist.github.com/Tomasz-W/911d68c20f95320e1e5f97bccf90114d

**Comment by Hufkratzer** [comment]
@Tomasz-W Thanks, we can continue with your drafts, I will have a closer look at them.

But you exchanged **equestrian** and **horse_riding**. Was that on purpose? I propose to use the jumper icon for sport=equestrian like it is in JOSM. (This has already been discussed in these posts above: [1](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-357177704) & [2](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-357300647).)

**Comment by Tomasz-W** [comment]
Sorry for the switch, I was uploading these icons on the evening, when I was quite tired, of course jumper one is for equestrian and the second one is for horse_riding.

When you will work on these icons in Inkscape, remember about trying to pixel align certain shapes to the grid (but it's not the priority here, because they are quite complicated icons), it makes icon sharper on the map.  When you finish, export designs as 14x14px *.png files and post them here.

**Comment by Hufkratzer** [comment]
@Tomasz-W 

[Here](https://gist.github.com/Hufkratzer/452a3aca0acce03bb4f1e52ce5d4121c) is my second draft for the horse icons, based on what you did.

The 14px exports look like:
![equestrian14](https://user-images.githubusercontent.com/5958184/45620218-71a2b600-ba7c-11e8-859e-c120031cc039.png) for sport=equestrian
![horse_riding14](https://user-images.githubusercontent.com/5958184/45620250-88e1a380-ba7c-11e8-84fd-2724fbd2cddb.png) for leisure=horse_riding
The black icons (on the left) are yours, the brown icons (on the right) my.

The same a bit larger and with grid:
![equestrian-grid140](https://user-images.githubusercontent.com/5958184/45620627-d1e62780-ba7d-11e8-98e9-791cada1fa0a.png)
![horse_riding-grid140](https://user-images.githubusercontent.com/5958184/45620725-1d98d100-ba7e-11e8-8294-446c6cca2ae8.jpg)

**Changes:**

for sport=equestrian:
* head and neck of horse lower, more like in photos given in post [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-415979354)
* hooves and nose of horse put in center of pixel, lower part of horses front leg longer
* obstacle lower, compare photos, horse will hardly get over the high one in this manner
* riders head smaller, should be smaller than horses one 
* horsetail leaner

for leisure=horse_ridng:
* rider leaner, looked really fat. Yes, pixel aligned lines look less blurry, but nevertheless the proportions should not be distorted too much IMHO; compare [picnic.svg](https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/tourism/picnic.svg), the icons should still look ok if scaled a bit larger, eg. in the wiki
* riders head smaller, should be smaller than horses one 
* hooves and nose of horse put in center of pixel
* horsetail leaner

Maybe I changed something that you did for some good reason just because I didn't know the reason; feel free to correct it.

**Comment by Tomasz-W** [comment]
Thanks! Looks ok for me. When you are posting Gist link with SVG file these steps should be done at the end:
* select all elements and click "path"->"union"
* delete all needless nodes (eg. when there is 3 or more nodes on a square/ rectangle side)
* save file as "Clean SVG"

You can download one of my SVGs and use it as template (delete my icon, then paste yours, do steps described above and save the file as clean SVG). It should be easier than changing your file settings.

You can wait with it for a few days, because some new suggestions may happen, and it's always annoying to save/ upload file again and again.

**Comment by Hufkratzer** [comment]
Here are the new horse icons:
![equestrian14](https://user-images.githubusercontent.com/5958184/45780381-c18ca300-bc5d-11e8-8dc2-371f7c9a865f.png) [equestrian.svg](https://gist.github.com/Hufkratzer/3177ebdd39abacfccc4cb171e89848d4#file-equestrian-svg)
![horse_riding14](https://user-images.githubusercontent.com/5958184/45780407-ccdfce80-bc5d-11e8-9b8a-8abde03f2eea.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/3177ebdd39abacfccc4cb171e89848d4#file-horse_riding-svg)

I couldn't find an option to save a file as "Clean SVG" in Inkscape 0.91, only "Normal SVG" what is without Inkscape-specific stuff, but still with other things that looked unnecessary; so I opened [picnic.svg](https://raw.githubusercontent.com/gravitystorm/openstreetmap-carto/master/symbols/tourism/picnic.svg) als a template in a text editor and replaced d= and id= with the ones of the horse icons. Seems to work this way.

**Comment by Tomasz-W** [comment]
I've fixed them a liitle bit (bigger heads, more pixel alligning)

![eque](https://user-images.githubusercontent.com/25656654/45798347-de4ec800-bca9-11e8-873c-3e61f771636b.png)
![leisure horse_riding](https://user-images.githubusercontent.com/25656654/45799095-9c735100-bcac-11e8-83c9-1ea84294e50e.png)

@Adamant36 It's ready for testing:
https://gist.github.com/Tomasz-W/911d68c20f95320e1e5f97bccf90114d

**Comment by Adamant36** [comment]
@Tomasz-W OK. I think I'm only going to do leisure=horse_riding though and save sport=equestrian for issue #844. Since @kocio-pl ocio-pl says its better not to mix things in a single PR in case one gets rejected. Plus, I'm still working on how to add a whole new rendering category to the code like is needed for the sports stuff. Maybe @kocio-pl can point me in the direction of some information on how to do that.

**Comment by Hufkratzer** [comment]
@Tomasz-W 
> I've fixed them a liitle bit (bigger heads, more pixel alligning)

**Extremely bad**, IMHO! Please explain the purpose of your "fixes" in more detail. How can an inclined line be more pixeö-aligned than a vertical one? I wonder why we discussed my drafts previously.

Here is your horse_riding icon (black) compared to my (brown):
![compare140](https://user-images.githubusercontent.com/5958184/45802935-7c498f00-bcb8-11e8-9308-44a6c7a134ec.png)

Your rider is much too big compared to the horse. He is also tilted forward without reason, was is clearly wrong and looks really awful, compare the photos [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-415979354).

If you explain what you want and why I can try to make a new version of icons.

**Comment by Tomasz-W** [comment]
Please don't compare icons in Inkscape scale, but in 14x14 scale, as we use them in OSM-Carto in this one. We shouldn't be "glued" to natural proportions of some objects, because sometimes it leads to lower readability of the icon. There have been voices that with vertical line it looked more like some "i" letter (see: https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-405082224), I also think it looked more like some stick than a human.

**Comment by Hufkratzer** [comment]
[#2344 (comment)](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-405082224) :
> That Mapbox’s looks like ‘i’ on a horse — it’s caused by those white jodhpurs which additionally make the icon look two­‑colored instead of single­‑colored.

Please have look at this vidio, 3rd miunute:
https://www.youtube.com/watch?v=-w8As0gln6o
and at the photos [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-415979354)

**Comment by Adamant36** [comment]
Jodhpurs or not, the guy is still barely visible/looks like a stick in @Hufkratzer version. Also, in @Hufkratzer's version the rider looks sunk into the horses midsection. Whereas, @Tomasz-W''s version just looks more natural because the rider appears to be up a little more and would be more inclined to lean forward with the horse as he rides, instead of sitting straight up. Its more "kinetic" that way and gives the icon a sense of movement that is missing otherwise. Really, a person wouldn't be sitting up straight on the horse unless they are either standing still or riding for show, which is covered by the other icon. But "riding" insinuates movement and the icon should reflect it.

**Comment by Adamant36** [comment]
@Hufkratzer, I mostly agree with you. Except that the forward slant is not that profound and is still less then for the sport icon. No person unless they are a professional Spanish horse rider is going to be able to stay at a straight angle 100% of the time. They just don't have the muscles for it. So there will always inherently be a slight tilt to how they sit. In fact, in the second link you provided, the riders are positioned leaning slightly back due to putting the weight on their seat bones. As the article says "it is important to keep the upper body well back, in order to help keep the weight on the back of the seat-bones." So if anything @Tomasz-W's icon should be adjusted back more, but still not be straight up like your say. 

Anyway, whatever complaint you might have about the riders position in @Tomasz-W's version, at least it looks like a rider and not a stick. If you want to create a better icon in the same position that looks more like a person go for it, but at this point I think @Tomasz-W's is the best one. I'm perfectly willing to give your revised one a chance though if you create it.

**Comment by Hufkratzer** [comment]
@Adamant36 
Yes. they lean back a bit; it is better than leaning forwards. This can also been seen on some of the photos to that I linked [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-415979354). Here is another video where the riding instructor explains that the rider should sit straight, but when she rides she often leans back a bit: https://www.youtube.com/watch?v=vVzXdLm-Eyg

So I made a new version of the horse_riding icon with the rider leaning back a bit. The head is larger (imagine the rider wears a helmet) and its position is in front of the now inclined axis of the rider, like in the hiker icon above. I hope this solves the "stick problem". Both, horse and rider, have become larger (horse higher, rider taller) to make the icon square.

The jumper icon has been adapted, so that both icons again show horse and rider in similar sizes, just doing different things. 

14px (left/black @Tomasz-W's, right/brown my icons):
![horse-icons14](https://user-images.githubusercontent.com/5958184/45931087-d1abc780-bf68-11e8-85a3-b7fb86acf00d.png)

double size:
![horse-icons28](https://user-images.githubusercontent.com/5958184/45931090-e12b1080-bf68-11e8-9b9f-43218ff596c2.png)

This is just a [draft for discussion](https://gist.github.com/Hufkratzer/1b53c69f2afbf35c48bf594e1bf21034).

IMHO the readability is the same in both versions and, of course, I would prefer to see icons on the map that show riders in correct postures.

**Comment by Tomasz-W** [comment]
@Adamant36 The best solution in situations like here is to make test renderings with both versions of icons  to not ignore anybody, and then decide. There are some things which we can approve/ reject with BW 14x14 images only, but finally we always need to rate how would it look filled by certain colour throwed on a map. I provided my designs in https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-423053362, if you need some files from @Hufkratzer , ask him.

**Comment by Adamant36** [comment]
@hufkratzer, how do you know doing test renderings wont make the icons look different? And why are us testing them? It helps to seem them in context, outside of a tiny green square. Plus, mapnik puts its own color filters on icons that make them look different when rendered. Why not just provide the .svg files? Otherwise, I'm not going to have an opinion different then I already had. Yeah, the new icon is an improvement, but that still doesnt mean its better then @Tomasz-Ws and we have no way to decide unless its test rendered first. Nore will we, because its almost a requirment. There wont be further discussion until then and more then likely we will just go Tomasz-Ws icon eventually if no alternative is provided. So, just get it in a state that you think is good enough, provide the SVG files of your individual icons, and we will go from there. There's no reason you cant just modify them later if need be.

**Comment by meased** [comment]
The horse jumping the gates looks like a carousel horse to me.

**Comment by Hufkratzer** [comment]
This with the carousel can be seen like that and this should be avoided. I made a draft (14px export) with thicker obstacles for discussion:
![caroussel14v4](https://user-images.githubusercontent.com/5958184/46575193-f5cbc780-c9b0-11e8-805a-a65201cf9159.png)
I think at least the one on the right can hardly be confused with a carousel horse.

I think @imagico may not like the horseshoe pattern for equestrian sport because something like this was already discussed some time ago [here](https://github.com/gravitystorm/openstreetmap-carto/issues/844#issuecomment-276781250) (you can insert "horseshoe" here instead of "horse"). I think the jumper expresses quite well that the icon is about equestrian **sport** (what else could be used to do that?) and if all the other pitches get an icon it may be irritating if just the one for sport=equestrian gets a pattern instead. Perhaps the horseshoe pattern can rather be used for leisure=horse_riding (in analogy to leisure=dog_park)?

**Comment by Adamant36** [comment]
I think the one on the right (the fatter one correct?) is an improvement, because it looks like more a cone being jumped over then a carousel horse. Maybe it could be shrunk down on the top a tad though to create some space or something. Also, The tail and the back leg looks to similar and like a crabs claw. Plus, the way the head is it  looks like the person is wearing a helmet that sticks out in the back (am I nitpicking? Probably, but that's how I feel like being). Finally, it would be good if they were in SVG format like they should be. Since it could look different that way, with the way it handles filters etc. Plus, as individual icons instead of side by side. Draft or not., that's the proper way to do it, as outlined in the contributing guidelines. As I have said already. 

Oh yeah, I don't think a horseshoe pattern would be a good idea. As it might be confused with a horse shoe game area. Plus, patterns are a pain to implement anyway.

**Comment by Hufkratzer** [comment]
Here is a new draft of the jumper icon for your test renderings if you like:
![equestrian14](https://user-images.githubusercontent.com/5958184/46587639-7a8d1300-ca8f-11e8-83f7-4610885c1d25.png) [equestrian.svg](https://gist.github.com/Hufkratzer/0eb09dbd0f547f00b33c2c6d1de17576/c4e7984da55417ac583c51d2d58dbc0296a81452#file-equestrian-svg)

The obstacle is really low now, looks a bit like in the title photo of
https://wiki.openstreetmap.org/wiki/Tag:sport=equestrian

I hope the horsetail is better now. I didn't understand the complaint about the helmet; if the rider sits on the horse leaning forward that much and wants to look straight forward (what he should) he has to lift his head.

It's a draft, but in case there are no more complaints, I think it could be used without further changes.

With the horse_riding icon I will not continue, it's a waste of time (for both of us). For test renderings that show the context you have @Tomasz-W's version.

**Comment by Adamant36** [comment]
@Hufkratzer, thanks. I appreciate you doing that. It looks a lot better now in my opinion. It also looks more like the picture in the wiki. Which was what I was going off of.

**Comment by Adamant36** [comment]
Any updates on this? Calling it a draft and then not working anymore kind of makes it not a draft. Id like to do PR on it and get it through before the sports icons. 

@Tomasz-W, any opinions? Do you think @Hufkratzer's last version is good enough to go with?

**Comment by Hufkratzer** [comment]
@Adamant36 
What are you complaining about? I don't understand. Is it about the equestrian or about the horse_riding  icon?

**Comment by Adamant36** [comment]
Where am I complaining? I was just asking what comment Tomasz-Ws icon was in so I can test both of them. We do that around here.

**Comment by Hufkratzer** [comment]
@Adamant36 
>Calling it a draft and then not working anymore kind of makes it not a draft

sounded like a complaint to me.

**Comment by Adamant36** [comment]
Oh yeah, I can see that. I was just commenting on the fact that you said it was a draft and then didn't do any more work on it. Kind of like a "nudge nudge" thing. So you'd come back and say if you were done with it or not.

**Comment by Hufkratzer** [comment]
You can take the **equestian** icon from [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-427694146)

Since meanwhile there was new thumb up [here](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-423077373) I now made a new file for 
![horse_riding14](https://user-images.githubusercontent.com/5958184/49373892-edb5ad00-f6ff-11e8-828a-51941b5b59ea.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/f78f1fd23499c59715d499329735c66c) with minor changes from my [last draft](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-423834213)

If anyone has suggestions for improvements I will see what I can do.

**Comment by Adamant36** [comment]
I hate the legs. They look to much like a saw horse. Its to symmetrical. Maybe that's why they are called that though.

**Comment by Hufkratzer** [comment]
I will make an icon with a walking horse instead of a trotting one if you think that may help, it will take a bit of time.

**Comment by Adamant36** [comment]
Hufkratzer's icon 
![hufkratzer](https://user-images.githubusercontent.com/30259065/50122273-83792e00-0210-11e9-9429-e79a796b2d98.png)
Tomasz-W's icon
![tomasz-w](https://user-images.githubusercontent.com/30259065/50122427-4e211000-0211-11e9-9f11-901c787f0111.png)

@Tomasz-W, area leisure green fill or pitch fill? I thought I had coded it in already, but I guess not. So I thought id double check.

**Comment by Adamant36** [comment]
Its cool. Its barely noticeable and isn't different then Tomasz-W's icon anyway. The only difference I can tell between the two is that yours a little cleaner looking and the legs are slightly more proportional. So I vote for yours as is. Unless @Tomasz-W wants to do a little more work on his or something. I'm fine waiting.

**Comment by Tomasz-W** [comment]
@Adamant36 
* area leisure greeen fill as it's kind of sport centre
* I'm ok with Hufkratzer's icon

**Comment by Hufkratzer** [comment]
@Adamant36
A new version with a walking horse:
![horse_riding14](https://user-images.githubusercontent.com/5958184/50496546-0714f500-0a31-11e9-82e0-414e1189fbc3.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/0c313c1a4dda30fc9c82ad7c8f9bacd6)
inspired by https://commons.wikimedia.org/wiki/File:The_horse_in_motion_as_shown_by_instantaneous_photography_BHL19590462.jpg, with less symmetrical legs than in the previous version (as requested [here](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-448031139)). Is this better?

Why didn't you test-render the equestrian icon(s) yet, neither here nor in the [PR 3578](https://github.com/gravitystorm/openstreetmap-carto/pull/3578)?

**Comment by Adamant36** [comment]
Thanks. I think the other icon is fine though. The back legs on this one look weird and the other one is fine. 

Did you check the sport PR? It has its own issue that need to be worked out. Equestrian is part of the sport icons. So I'm handling it separately. As it takes different things on the coding side and is a separate issue. I'm still going to get to it eventually though.

**Comment by Hufkratzer** [comment]
https://github.com/gravitystorm/openstreetmap-carto/pull/3578 is for pitches ans the equestrian icon **is** for pitches too, like the tennis icon, see 
https://wiki.openstreetmap.org/wiki/Tag:leisure=pitch#Riding_arena

I will try to improve the legs of the horse_riding icon.

**Comment by Adamant36** [comment]
Rendering with areas for anyone that's interested. The icon starts displaying at z17. Outlines were never discussed so I didn't add them. Personally, I don't feel like the area needs an outline because it usually has fences that can be mapped, although I guess I can always if there's a need for it. I'll probably do a PR on it tomorrow. Feel free to provide any feedback in meantime. 
![horse riding 2](https://user-images.githubusercontent.com/30259065/51798822-8252fd80-21cd-11e9-88e0-b6719569423b.png)
![horse riding](https://user-images.githubusercontent.com/30259065/51798785-c09bed00-21cc-11e9-8c8d-a93a99f85968.png)

**Comment by Hufkratzer** [comment]
I could not make a good-looking icon with very asymmetrical legs, I can only  offer a 2nd version with a walking horse who has a bit asymmetrical legs that look less like saw horses:
![horse_riding14](https://user-images.githubusercontent.com/5958184/51807499-cd314b80-2287-11e9-8c41-2c2931e27466.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/b54b7d5aa3d1a457e83fd2ef269ccb69), compare illustration in http://swphorses.com/understanding-horses-gait-part-2/

I you nevertheless prefer the trotting horse I can offer a new version with improved pixel-alignment:
![horse_riding14](https://user-images.githubusercontent.com/5958184/51807565-70826080-2288-11e9-9d25-6339c34019aa.png) [horse_riding.svg](https://gist.github.com/Hufkratzer/573e8830a90e2ecbdcb9f0898967b97d)

Here is a comparison of these 2 icons (black and green) and the [previous trotting version](https://gist.github.com/Hufkratzer/f78f1fd23499c59715d499329735c66c) (red):
![comparison400](https://user-images.githubusercontent.com/5958184/51807587-e1c21380-2288-11e9-8528-db6b7d5d12df.png)

Additionally the new icons no longer have a 14px canvas because [here](https://github.com/gravitystorm/openstreetmap-carto/pull/3578#issuecomment-450810553) @Tomasz-W recommended to use a sample without one.

**Comment by Adamant36** [comment]
Thanks. Your original icon that I used in the sample rendering is fine. Plus its the one other people approved. So I think I'm just going with it. I don't want to nitpick over a one or two pixel difference here or there.

**Comment by Nic787** [comment]
What happened with this project? The wiki say to name farmyard instead, but when you don’t know the limit of the horse facility, it’s better using a node leisure=horse_riding. While you can search the name via the search bar, it just doesn’t render.

**Comment by Hufkratzer** [comment]
3 weeks ago there was a request to "revisit the topic" in https://github.com/gravitystorm/openstreetmap-carto/pull/3660#issuecomment-806067383.

To use landuse=farmyard is (AFAIKS) only recommended on the German page   [DE:Tag:leisure=horse_riding](https://wiki.openstreetmap.org/wiki/DE:Tag:leisure%3Dhorse_riding). I think logically this only makes sense for places that do not mainly keep horses for riding but mainly do other farming activities because leisure=horse_riding is more specific. But it is probably often used additionally because leisure=horse_riding is not rendered yet and landuse=farmyard is (see also https://github.com/gravitystorm/openstreetmap-carto/pull/3660#issuecomment-458274130). In Germany 9.3 % of the horse_riding places also have tag landuse=farmyard, in the whole world 5.7 %. Should this recommendation about landuse=farmyard better be removed from the German wiki page?

**Comment by Adamant36** [comment]
> Should this recommendation about landuse=farmyard better be removed from the German wiki page?

I'd so no. horse riding facilities can be part of a farmyard. Maybe change it to sound more like an optional thing that should be tagged on a bigger area then just the horse riding facility if it's not phrased that way already.

**Comment by Hufkratzer** [comment]
On a bigger area ok, I think this goes without saying. But I suspect that putting landuse=farmyard and leisure=horse_riding on the same area element, as it is probably done because horse_riding is not rendered yet, violates the [One feature, one OSM element](https://wiki.openstreetmap.org/wiki/One_feature,_one_OSM_element) rule. One area can only be rendered either as farmyard or as horse riding facility, not as both. 

On the German page it is said that landuse=farmyard should be used for professional operations ("Profi-Betriebe"), but leaving open what a "professional operation" shall be. Is there any reason why a place where someone professionally keeps horses and offers trails rides should be tagged with landuse=farmyard rather than with leisure=horse_riding?

**Comment by dieterdreist** [comment]
Am Do., 15. Apr. 2021 um 19:25 Uhr schrieb Hufkratzer <
***@***.***>:

> On a bigger area ok, I think this goes without saying. But I suspect that
> putting landuse=farmyard and leisure=horse_riding on the same area element,
> as it is probably done because horse_riding is not rendered yet, violates
> the One feature, one OSM element
> <https://wiki.openstreetmap.org/wiki/One_feature,_one_OSM_element> rule.
> One area can only be rendered either as farmyard or as horse riding
> facility, not as both.
>

landuse=farmyard is not a tag for a farmyard, it is a property on the land,
saying it has farmyard usage. The wiki puts it like this: "Area of land
with farm buildings (farmhouse, sheds, stables, barns, etc.)"
This is not conflicting with leisure=horse_riding (which seems a bit ill as
a tag, because horse riding is an activity, while leisure usually describes
the kind of physical place).



> On the German page it is said that landuse=farmyard should be used for
> professional operations ("Profi-Betriebe"), but leaving open what a
> "professional operation" shall be. Is there any reason why a place where
> someone professionally keeps horses and offers trails rides should be
> tagged with landuse=farmyard rather than with leisure=horse_riding?
>

if you discovered that the German version differs from the original version
you should mention it on the German forum, so it can be discussed and fixed.

**Comment by Adamant36** [comment]
> On the German page it is said that landuse=farmyard should be used for professional operations ("Profi-Betriebe"), but leaving open what a "professional operation" shall be. Is there any reason why a place where someone professionally keeps horses and offers trails rides should be tagged with landuse=farmyard rather than with leisure=horse_riding?

As far as I know the place where someone keeps the horses is tagged as building=barn and with the trails, it's more of an access tagging thing. Which I think goes to show what @dieterdreist is talking about. The tag seems kind of vague and not clear when or how to use it. It's not really a leisure thing if you confined it to "professional operations" is it? What would that mean and how would you determine it anyway?   

BTW, the English article has the same lack of clarity and I don't think things can be solved just by changing the wording of the German article.

**Comment by Hufkratzer** [comment]
@Adamant36 , sorry, I don't understand you  If [landuse=farmyard](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dfarmyard) can be used in parallel with [leisure=horse_riding](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dhorse_riding) I see no problem anymore. The operation may be professional (or not) but the people who use it do that in their spare time (at least most of them), like in a [leisure=fitness_centre](https://wiki.openstreetmap.org/wiki/Tag%3Aleisure%3Dfitness_centre) for example. Meanwhile I have also found that @dieterdreist [has explained it this way in a recent discussion in the German forum](https://forum.openstreetmap.org/viewtopic.php?pid=817499#p817499) . So what exactly is "vague and not clear" in the [English article](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dhorse_riding)? I would just add [landuse=farmyard](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dfarmyard) under **See also** there.

BTW, buildings where horses are being kept should be tagged with [building=stable](https://wiki.openstreetmap.org/wiki/Tag:building%3Dstable) (But because horses tend to live outside the whole place usually contains more than just a stable).

**Comment by Adamant36** [comment]
> @Adamant36 , sorry, I don't understand you If landuse=farmyard can be used in parallel with leisure=horse_riding I see no problem anymore. The operation may be professional (or not) but the people who use it do that in their spare time (at least most of them), like in a leisure=fitness_centre for example. Meanwhile I have also found that @dieterdreist has explained it this way in a recent discussion in the German forum . So what exactly is "vague and not clear" in the English article? I would just add landuse=farmyard under See also there.

The problem with the comparison to the leisure=fitness_centre tag is that fitness centers are contained in a building and there is no ambiguity about what constitutes one or it's boundaries (usually the walls of the buildings it is contained in). That's not the case with horse riding facilities. 

The English Wiki article says "The tag is for the whole facility which may include riding arenas, stables, riding halls, other buildings, paddocks and more (compare also Riding)." So, he tag could literally be applied to anything where there's someone leisurely riding a horse. Especially with the whole "and more" part at the end. Compare that to leisure=fitness_centre's article, "They are places to go to for exercise. Otherwise known as a gym, health club, or fitness club." There's no ambiguity there about what's being tagged or when to use it. 

Say you have just a barn, is that a riding facility? According to tags article it is. Even though you aren't riding the horse in a barn. How far down the trails do you map the area? A few miles because that's how far you can leisurely ride your horse? Do you include the house next to the barn? What about the front lawn or the driveway? When is something a "riding facility" and not just a farm where people have a few horses that they ride around sometimes? Everyone knows what a gym is. 

We shouldn't encouraging the usage of vague tags. Especially when there is leisure=track + sport=equestrian and there's zero reason not to just use that tag for what's clearly being used exclusively for horse riding. The tag isn't being used consistently anyway. From the research I've done it is being used on a hodgepodge of stuff that isn't at all consistent. For instance, it's used in some places where building=barn or leisure=track + sport=equestrian  should be used instead. So, whatever your understanding of it is, it's clearly not being that way. 

That and I'm pretty sure we have had this conversation a few times now over the last couple of years and I've still not heard a good argument for why it should it be rendered or used instead of the other tags that cover the same things. In otherward, what's unique about that can't and aren't already being mapped with other tags?

**Comment by Hufkratzer** [comment]
The current description on page [leisure=horse_riding](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dhorse_riding) is in its style similar to the description on page [leisure=sports_cemtre](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dsports_centre), just adapted to riding stuff.

Since it is often more difficult to determine the boundaries for horse riding facilities than for sports centres, especially for indoor sports centres, it would be desirable if a node with leisure=horse_riding would be rendered somehow (see comment by @Nic787  [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-817322241)). Of course the trails / streets / houses in the surroundings, where the people hack out, do not belong to the horse riding facility, except they are on the property of the facility.

Do you really think that the area that is tagged with leisure=horse_riding in the [example](https://www.openstreetmap.org/way/485444067) on the horse_riding page should better be tagged with [leisure=track](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dtrack) + sport=equestrian and that this would "be way more descriptive and better understood by mappers"?

**Comment by Adamant36** [comment]
> The current description on page leisure=horse_riding is in its style similar to the description on page leisure=sports_cemtre, just adapted to riding stuff.

Not really, because leisure=sports_centre article is clear in the first few sentences that it's meant for "a distinct facility where sports take place **within an enclosed area**." It's also pretty clearly gives details on what those facilities are and doesn't say "or whatever" at the end of the description. Whereas, the leisure=horse_riding article doesn't say anything that specific. Only "where people practice horse riding." Which could literally be anything anywhere. 

Sure, it might say the tag applies "may include riding arenas, stables, riding halls, other buildings", but if your using it on any of those things your either leaving out better, more descriptive tags or using leisure=horse_riding as an alternative tagging scheme. Which is the only ways (except a few times) that I've seen the tag used. 

> Since it is often more difficult to determine the boundaries for horse riding facilities than for sports centres, especially for indoor sports centres, it would be desirable if a node with leisure=horse_riding would be rendered somehow

I'd be perfectly fine with that. Except it doesn't negate or address the problems that I've brought several times of it mainly either being used in place of better tags or as a an alternative/co tagging scheme for a bunch of other tags. Which we shouldn't be doing. Let alone encouraging by rendering the tag. 

Also, by encouraging the tagging of a node on something that has "difficult to determine boundaries" you would be getting into the same issues there are with the rendering of bodies of water that don't have clear middle points or boundaries, and rendering them has recently been removed for that reason. So, I doubt node rendering will be approved. Really it shouldn't be. 

> Of course the trails / streets / houses in the surroundings, where the people hack out, do not belong to the horse riding facility, except they are on the property of the facility.

If there on the property of the horse riding facility don't they belong to it then? There in lays the problem I guess, because half the people using the tag would assume they do and half wouldn't. Clearly there is no good measure of what is or isn't part of the horse riding facility except for whatever the person feels like saying is or isn't a part of it at the time. 

> Do you really think that the area that is tagged with leisure=horse_riding in the example on the horse_riding page should better be tagged with leisure=track + sport=equestrian and that this would "be way more descriptive and better understood by mappers"?

No. I don't think the leisure=horse_riding tag is necessary there because all the actual, physical, on the ground elements are already appropriately tagged in a way that makes it clear that it is a horse riding facility. Not to mention the name of the place makes it clear that's what is to. There's zero to add leisure=horse_riding to anything at that point. Except for useless duplication and having an alternative tagging scheme for it's own sake. I'd love to see an example of any other tag that is used that way. 

BTW, there's a park near me that has a few horse stables for day use and trails where people ride their horses around. They do the rodeo there once a year to. I'd love to know how you'd tag that place as a horse riding facility, let alone why it should be tagged that way at all, when it's already a mesh of other stuff that is tagged to show horses are permissible there and you sure can't add leisure=horse_riding to the areas that are already tagged as leisure=park. Like you say there is no clear boundary of where the horse usable parts and the non-horse usable are either. It's pretty much whatever people feel like riding their horses on or not and a lot depends on when in the year it is. So there's really no way to tag it as a horse riding facility. Even though people the general, philosophical sense that it is used for that sometimes. 

My point is, it's not something as clear cut and easy to define as something like landuse=retail or other leisure examples.

**Comment by imagico** [comment]
Following up on https://github.com/gravitystorm/openstreetmap-carto/issues/3953#issuecomment-973963209 - can someone familiar with the domain explain to me why https://www.openstreetmap.org/way/203993861 is tagged leisure=horse_riding and not leisure=sport_centre + sport=equestrian?

**Comment by ReinerMeyer** [comment]
imagico has forwarded my request to here (Following up on #3953 (comment)).
Here are a few additions to better understand my request:
I am not familiar with horses, have never ridden. Nevertheless, my hair stands on end if I should describe horse_riding (animal-human relationship) as farmyard (farm for obtaining food) or sport_centre (human-centered sporting activities). They are 3 different worlds to me.

For me, classifying horse_riding as a separate tag is comparable to tagging for dogs. Dogs and horses have a special meaning for us humans.

**Comment by imagico** [comment]
I don't intend to discuss how certain horse related features *should be* tagged.  I am interested in understanding how things *are* tagged.  So my question remains - why is *Reitstall Wiener Neudorf* tagged leisure=horse_riding and not leisure=sport_centre + sport=equestrian.  Back in 2019 (https://github.com/gravitystorm/openstreetmap-carto/pull/3660#issuecomment-458105379) i determined that the tag is used with very poor in consistency of use and overlaps massively with other much more common tags.  But things develop and i want to keep an open mind so i ask those more familiar with the field to explain to me the delineations between the different tags in practical use based on concrete examples.  Is leisure=sport_centre + sport=equestrian considered a deprecated tagging (i.e. not to be used for non-racing sports with horses)?  Or is leisure=sport_centre + sport=equestrian only used for facilities predominantly serving competitive equestrianism while facilities serving [pleasure riding](https://en.wikipedia.org/wiki/Pleasure_riding) are tagged leisure=horse_riding?

**Comment by ReinerMeyer** [comment]
There is a larger area near Ebreichsdorf where leisure=sport_centre really applies (node: Magna Racino Ebreichsdorf (920147244)). This area contains beside the racetrack also parking lots and stables. Unfortunately, only one node was used for this.

The area mentioned by me (way: Reitstall Wiener Neudorf (203993861)) but also many other - further example (way: Reit- und Therapiezentrum Kottingbrunn (180670645)) - correspond rather to the pleasure riding (https://en.wikipedia.org/wiki/Pleasure_riding) corresponds.
The fact that you can see the name of the area (way: Reit- und Therapiezentrum Kottingbrunn (180670645)) is due to the fact that the complete area was also tagged as landuse=meadow. This makeshift should not be necessary.
This horse_riding near Kottingbrunn is also used for therapeutic purposes.

The golf course near Oberwaltersdorf (path: Golfclub Fontana (30551552)) is more comparable to the leisure=sport_centre than the two horse_riding I mentioned as examples.

**Comment by Hufkratzer** [comment]
@imagico 
> Is leisure=sport_centre + sport=equestrian considered a deprecated tagging (i.e. not to be used for non-racing sports with horses)?

It is not deprecated, but it is not documented in the wiki anymore (see [here](https://wiki.openstreetmap.org/w/index.php?title=Tag%3Aleisure%3Dsports_centre&type=revision&diff=1790054&oldid=1774697) and [here](https://wiki.openstreetmap.org/w/index.php?title=Tag%3Aleisure%3Dsports_centre&type=revision&diff=1901075&oldid=1901031)).

It is also generally used very little compared to leisure = horse_riding, see
<img width="421" alt="statistics" src="https://user-images.githubusercontent.com/5958184/142924117-406859f7-3f7d-4e50-ad6d-257a06aaf237.png">

In https://github.com/gravitystorm/openstreetmap-carto/pull/3660#issuecomment-458304955 you wrote:
>  the very idea to distinguish between non-sport leisure riding and sport leisure riding is nothing that is likely going to work in OSM on a global level since definitions and understanding of sports vary too much between different cultures.

If this is nothing that is going to work, I would propose to use leisure=horse_riding also for places where mostly non-racing sports with horses is practised; in these cases sport=equestrian can be added to leisure=horse_riding (while leisure=sports_centre without sport=equestrian is too general for non-sport leisure riding). But we also have 1k sports_centre + sport=9pin or 10pin "competing" with 4k bowling_alley, they are all rendered and I don't know why this is a problem. Compare this [wiki discussion](https://wiki.openstreetmap.org/w/index.php?title=Talk:Tag:leisure%3Dhorse_riding#When_to_use_leisure.3Dhorse_riding_instead_of_leisure.3Dsports_centre.3F) where (at the end) you can also find some examples where I would consider leisure=sports_centre inappropriate.

Regarding the issue that many small areas tagged with leisure=horse_riding look more like riding arenas in the aerial view and possibly should better be tagged with pitch + equestrian, I got an **idea**: They mostly have no name, but a real horse riding facility that deserves this tag should have one. Is it possible to render only leisure=horse_riding + name=* (and not the ones without a name? 6.7k (47%) have a name.

**Comment by imagico** [comment]
We won't make decisions on non-label rendering of features depend on the presence of a name tag - that would be incompatible with our goal to provide meaningful and intuitive mapper feedback.

My impression at the moment is that there is no clear consensus on how different tags for horse related infrastructure are delineated towards each other.  If that is the case we would need to be very careful with rendering decisions.  If we'd - as this issue suggests - selectively render leisure=horse_riding with a generic horse riding symbology that would communicate to the mappers that this is the tag to be used for any and all facilities where horses are kept for riding, including many horse farms or for example the stables and training areas for horses of mounted police units.  I am not sure we'd do anyone a favor with that.

The absolute numbers of tag use are not necessarily that meaningful because you would (a) have to take into account the ratio of prevalence of the different types of facilities in the countries where most of the mapping takes place and (b) you need to consider that leisure=sports_centre might also be used without sport=equestrian - either due to incomplete tagging or because it is a facility which serves also other sports.

As i said elsewhere i would like to see us rendering more animal keeping related features in a differentiated fashion because those in many parts of the world have high significance for peoples' everyday lives.  But we depend on mappers developing clear and viable tagging systems for those, especially in light of the fact that animal keeping practices varies a lot around the world.

**Comment by Adamant36** [comment]
There's also some instances of leisure=pitch + sport=equestrian out there. I haven't looked into how many or the specific usage, but it at least shows tagging is all over the place currently.

**Comment by ReinerMeyer** [comment]
If horses are used as draught horses, there is no tag for them.
Horses are probably mainly used for riding, so the tag horse_riding is actually always correct, regardless of whether they are used for, sporting competitions or simply for the pleasure of riding.
For sport competitions the tag combination leisure=sport_centre + sport= equestrian is suitable, for police operations the tag combination leisure=horse_riding + amenity=police or operator=police is suitable.

**Comment by Hufkratzer** [comment]
@ReinerMeyer 
> for police operations the tag combination leisure=horse_riding + amenity=police or operator=police is suitable.

Hm, from [wiki/Key:leisure](https://wiki.openstreetmap.org/wiki/Key:leisure):
> The leisure key is for places people go in their spare time. See also sport=* and tourism=*.

**Comment by ReinerMeyer** [comment]
It may possible that the tag leisure=horse_riding goes into the wrong direction.
In German, I would rather call the area a horse farm (Pferdehof) or riding farm (Reithof). These terms also often appear in the name of the facility. Perhaps using the tag landuse would be better (landuse=horse_farm or landuse=riding_farm or landuse=horse_riding).

A major difference from landuse=farmyard would be the type of use. Farmyard is a homestead for the production of food. A horse/riding farm is a service facility.

**Comment by Hufkratzer** [comment]
Oh, I think this discussion gets too complicated for here ([once again](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-248590270)) and should better be continued on the tagging mailing list or on  [wiki/Talk:Tag:leisure=horse_riding](https://wiki.openstreetmap.org/wiki/Talk:Tag:leisure=horse_riding).

I don't think that the leisure key is a major issue, it also isn't for leisure=pitch. But perhaps the wiki description can be improved somehow (see criticism in previous posts, [here](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-821940014) and [here](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-976941473)). 

Regarding farmyard or not, there was a [discussion in the German forum](https://forum.openstreetmap.org/viewtopic.php?pid=817499) (already mentioned [above](https://github.com/gravitystorm/openstreetmap-carto/issues/2344#issuecomment-821816301))

**Comment by ReinerMeyer** [comment]
The occasion of my discussion contribution was the lack of understanding why a registered name for leisure=horse_riding is not displayed in the official OSM map, especially since this non-display is explicit for this tag.
The older and current discussion posts have been valuable to me (also the links therein), but have not been able to clarify for me why this explicit exclusion exists. I suspect that I have been too little involved in the whole history. I hope that there will be a solution very soon, so that there will be no more tricking necessary with other tags. Mappers are very inventive there.

I have no other ideas on how to solve the dilemma. I guess the horse experts should come in.

**Comment by imagico** [comment]
> but have not been able to clarify for me why this explicit exclusion exists

I have tried to explain that in https://github.com/gravitystorm/openstreetmap-carto/issues/3953#issuecomment-973995075 - we render tags, not keys.  There is no *explicit exclusion*.  We don't currently render `leisure=horse_riding` for the reasons explained here and in #3660.

**Comment by Adamant36** [comment]
> I hope that there will be a solution very soon, so that there will be no more tricking necessary with other tags.

What other tags are being used? From the research I've done no one is tricking anything with other tags. It's just that how to use leisure=horse_riding isn't clear so people are using tags that describe the object better. That's how OSM works. 

> I have no other ideas on how to solve the dilemma. I guess the horse experts should come in.

Taking it to the tagging mailing list is a good suggestion. It's ultimately not on the maintainers of the style to work these types of issues out or pick "favorites." It's probably pointless to ask about it on the leisure=horse_riding talk page. There really needs to be a wider discussion and more authoritative about this then the Wiki allows for.

**Comment by Hufkratzer** [comment]
Of course, the wiki discussion pages can also be used to discuss everything; it is done everywhere all the time, it is completely normal.

**Comment by Adamant36** [comment]
> Of course, the wiki discussion pages can also be used to discuss everything; it is done everywhere all the time, it is completely normal.

I don't really care either way, but the Wiki usually gets a lot less participation then the tagging list does and the more opinions about it the better. I'd also hate to see two random people saying something on the Wiki being treated like it's authoritative if that's all the comments you get.

**Comment by pyrog** [comment]
Sorry, I don't read all posts in this long discussion 😅

Is it possible to render `leisure=horse_riding` in 2022 ?
- At least like `leisure=sports_centre` + `sport=equestrian` : a green label.
- Better, render the horse icon 😃 
- And why not, a different icon for both tags 😎

**Comment by martino2002** [comment]
I would like to add myself as a person who supports the creation of a way to render the tag leisure=horse_riding.
